docs/user/monitor_experiments_with_tensorboard.md

```diff
@@ -0,0 +1,178 @@
+
+logger.log(tabular)
+```
+
+## The "Automatic" Way: Setup Your Experiment with `@wrap_experiment`
+Garage provides a decorator `@wrap_experiment` to help the logger setup process.
+
+Suppose you have an experiment function `my_experiment()` with the following definition:
+```bash
+def my_experiment(seed=1):
+```
+Here's how you can enable logging for this experiment:
+
+First, import the `wrap_experiment` decorator function.
+```bash
+from garage import wrap_experiment
+```
+
+Now wrap your function:
+```bash
+@wrap_experiment
+def my_experiment(ctxt=None, seed=1):
+    """
+    Args:
+        ctxt (garage.experiment.ExperimentContext): The experiment
+            configuration used by LocalRunner to create the snapshotter.
+    """
+```
+`@wrap_experiment` helps create experiment log directories and set up logger outputs (e.g. `logger.add_output(dowel.TensorBoardOutput(log_dir))`). After the setup process, `my_experiment()` is called to start the actual experiment.
+
+Note that `my_experiment()` now receives a new input `ctxt`. This is required for  the wrapped function to receive experiment configuration created by the wrapper.
+
+This decorater can also be invoked **with arguments**:
+
+```bash
+@wrap_experiment(log_dir='my_dir')
+def my_experiment(ctxt, seed):
+```
+For a complete list of arguments, you can refer to the definition of `wrap_experiment` [here](https://github.com/rlworkgroup/garage/blob/master/src/garage/experiment/experiment.py).
+
+Congrats! Now you have finished setting up TensorBoard logging. You can run your experiment and find your TensorBoard logs under the log directory.
+
+By default, your `log_dir` is `garage/data/local/experiment/{experiment_name}_{experiment_id}`
+
+For our example experiment `my_experiment`, the default log directory is `garage/data/local/experiment/my_experiment`.
+
+Now if you run the experiment again, the logs of the second run will be saved under `garage/data/local/experiment/my_experiment_1`.
+
+## View Your TensorBoard Logs
+
+You can view your TensorBoard results via the TensorBoard command line tool:
+
+`$ tensorboard --logdir logs`
+
+Once you run the command, you will see
+```bash
+$ tensorboard --logdir logs
+Serving TensorBoard on localhost; to expose to the network, use a proxy or pass --bind_all
+TensorBoard 2.2.2 at http://localhost:6006/ (Press CTRL+C to quit)
+```
+
+Open your browser and navigate to http://localhost:6006/ to see a dashboard of your TensorBoard results.
+
+Note that if your `--logdir` directory contains multiple TensorBoard logs, the dashboard will show all of them. This is useful when comparing results from multiple runs.
+
+You can find more TensorBoard dashboard details [here](https://www.tensorflow.org/tensorboard/get_started).
+
+## Share Your TensorBoard Logs
+
+[TensorBoard Dev](https://tensorboard.dev/) allows you to share your TensorBoard results with others.
+
+Upload your logs by:
+```bash
+$ tensorboard dev upload --logdir logs \
```

Comment:
probably use an example command here which is likely to work with one of the garage examples (e.g. logs are sent to `data/local`